src/parse.rs

```diff
@@ -373,13 +373,22 @@ impl fmt::Display for UnsignedDecimal {
 /// Bit string whose length is a power of two.
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub enum Bits {
-    /// Least significant bit of byte
+    /// Least significant bit of byte.
+    ///
+    /// The bit is saved as a [`u8`] value that is always less equal 1.
```

Comment:
In 980a894f9c05336579c91ab02ada69c4dc74fbb0

`less equal` should be `less than or equal to` or `<=` everywhere.

Reply:
Yeah, in English there is no real short way to say this. Often you will hear "no greater than" which is shorter than "less than or equal to" but still a bit of a mouthful.